src/vscode-bicep/package.json

```diff
@@ -234,6 +243,12 @@
         "key": "ctrl+k i",
         "mac": "cmd+k i",
         "when": "resourceLangId == bicep"
+      },
+      {
+        "command": "bicep.decompile",
```

Comment:
@puicchan Shortcut: CTRL+B, D      (B for Bicep)

Reply:
Actually, removed the shortcut because it's difficult to find one that doesn't conflict (and it's in JSON files, not bicep) and don't expect it to be a common command.